compose/docker-compose.yaml

```diff
@@ -7,7 +7,7 @@ services:
     networks:
        - net_clair
   clair:
-    image: arminc/clair-local-scan:v2.0.6
+    image: arminc/clair-local-scan:latest
```

Comment:
Won't this be dangerous?

Reply:
nope, it's desired. That clair-local-scan is prebuilt image which contains alert database nearly at latest state to start with. It's bit a special case.

If plain db is used it will take ~1h to get enough data (vulnerability database) to really get any meaningfull results. Data amount isn't that big, but it seems downloading data from central vulnerability database has been tweaked for pretty slowmo.